pkgs/tools/misc/conserver/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, autoreconfHook
+, gssapiSupport ? false
+, libkrb5
+, freeipmiSupport ? false
+, freeipmi
+, ipv6Support ? true
+, opensslSupport ? true
+, openssl
+, trustUdsCredSupport ? false
+, udsSupport ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "conserver";
+  version = "8.2.7";
+
+  src = fetchFromGitHub {
+    owner = "bstansell";
+    repo = "conserver";
+    rev = "v${version}";
+    sha256 = "sha256-LiCknqitBoa8E8rNMVgp1004CwkW8G4O5XGKe4NfZI8=";
+  };
+
+  # Remove upon next release since upstream is fixed
+  # https://github.com/bstansell/conserver/pull/82
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/bstansell/conserver/commit/84fc79a459e00dbc87b8cfc943c5045bfcc7aeeb.patch";
+      sha256 = "sha256:1dy8r9z7rv8512fl0rk5gi1vl02hnh7x0i6flvpcc13h6r6fhxyc";
+    })
+  ];
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  buildInputs = [ ]
+    ++ lib.optionals (freeipmiSupport) [ freeipmi ]
+    ++ lib.optionals (gssapiSupport) [ libkrb5 ]
+    ++ lib.optionals (opensslSupport) [ openssl ];
+
+  configureFlags = [ "--with-ccffile=/dev/null" "--with-cffile=/dev/null" ]
```

Comment:
I haven't looked at the makefiles, but is having both `--with-ccffile` and `--with-cffile` intentional?

Reply:
Yes it is, one is the client config and the other is the server config.  I called it out in an above comment: https://github.com/NixOS/nixpkgs/pull/181543#issuecomment-1186350878

But specifically:

```
Set --with-ccffile=/dev/null" "--with-cffile=/dev/null" so the default location wouldnt be $out/etc/conserver.cf or $out/etc/console.cf since a config is up to the user and can be passed with -C
```